### ISSUE TITLE
How can I train model using multi GPUS in a machine?

### ISSUE THREAD

**Issue by Kyfafyd** [kekmodel/FixMatch-pytorch#31]

**Comment by kekmodel** [comment]
example)
python -m torch.distributed.launch --nproc_per_node 4 ./train.py --dataset cifar100 --num-labeled 10000 --arch wideresnet --batch-size 16 --lr 0.03 --wdecay 0.001 --expand-labels --seed 5 --out results/cifar100@10000

**Comment by Kyfafyd** [comment]
Here, I have four GPUs in my machine, and run the command with original code from repo. But I got error like below, it seems the training iter will stack at 0 iter either. Can you help me figure it out? Thank you!
<img width="1440" alt="Screen Shot 2021-03-05 at 1 55 42 PM" src="https://user-images.githubusercontent.com/42490513/110073529-7c6f7380-7dba-11eb-8ac8-36eb91a816f1.png">

**Comment by Kyfafyd** [comment]
I wonder another question: Will distributed training faster than single GPU? If it is, how much? Besides, will distributed training  afffect the performance?

**Comment by kekmodel** [comment]
Please try running it again after 'git pull'.

**Comment by Kyfafyd** [comment]
I am running the up-to-date code.
But I still meet this error.

**Comment by kekmodel** [comment]
Sorry. Try again.

**Comment by Kyfafyd** [comment]
Thanks very much!
It works now
